src/kafka-util/src/client.rs

```diff
@@ -323,6 +329,57 @@ impl<C> TunnelingClientContext<C> {
     pub fn inner(&self) -> &C {
         &self.inner
     }
+
+    /// Returns a _consolidated_ `SshTunnelStatus` that communicates the status
+    /// of all active ssh tunnels `self` knows about.
+    pub async fn tunnel_statuses(&self) -> SshTunnelStatus {
```

Comment:
Since the tunnel status is consolidated, probably the method should be named `tunnel_status`?

```suggestion
    pub async fn tunnel_status(&self) -> SshTunnelStatus {
```